examples/openlayers/filtered.html

```diff
@@ -0,0 +1,154 @@
+<html>
+<head>
+    <link rel="stylesheet" href="/examples/site.css" />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.5.0/css/ol.css" type="text/css">
+    <script src="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.5.0/build/ol.js"></script>
+    <script src="https://unpkg.com/underscore@1.13.1/underscore-min.js"></script>
+    <script src="https://unpkg.com/flatgeobuf@3.7.1/"></script>
+</head>
+<body>
+    <ul class="primary-navigation">
+        <li>
+            <a href="/examples/leaflet/filtered.html">Leaflet Examples</a>
+        </li>
+        <li class="active">
+            OpenLayers Examples
+        </li>
+    </ul>
+    <ul class="secondary-navigation">
+        <li><a href="/examples/openlayers/">Basic Example</a></li>
+        <li class="active">Filter By Rect</li>
+        <li><a href="/examples/openlayers/large.html">Filtering a Large Dataset</a></li>
+    </ul>
+
+    <style>
+      #map {
+        width: 100%;
+        height:400px;
+      }
+    </style>
+
+    <div id="map"></div>
+    <div id="popup" class="ol-popup">
+      <a href="#" id="popup-closer" class="ol-popup-closer"></a>
+      <div id="popup-content"></div>
+    </div>
+    <script>
+        document.addEventListener("DOMContentLoaded", () => { 
+            // Elements that make up the popup.
+            var popupContainer = document.getElementById('popup');
+            var popupContent = document.getElementById('popup-content');
+            var popupCloser = document.getElementById('popup-closer');
+
+            // Create an overlay to anchor the popup to the map.
+            var overlay = new ol.Overlay({
+              element: popupContainer,
+            });
+
+            /**
+             * Add a click handler to hide the popup.
+             * @return {boolean} Don't follow the href.
+             */
+            popupCloser.onclick = function () {
+              overlay.setPosition(undefined);
+              popupCloser.blur();
+              return false;
+            };
+
+            // convert the rect into the format flatgeobuf expects
+            function fgbBoundingBox(centerCoords) {
+                let widthMeters = 500000;
+                const minPoint = new ol.geom.Point(centerCoords);
+                minPoint.translate(widthMeters/-2, widthMeters/-2);
+                minPoint.transform('EPSG:3857', 'EPSG:4326')
+                    
+                const maxPoint = new ol.geom.Point(centerCoords);
+                maxPoint.translate(widthMeters/2, widthMeters/2);
+                maxPoint.transform('EPSG:3857', 'EPSG:4326')
+
+                return {
+                    minX: minPoint.getCoordinates()[0],
+                    minY: minPoint.getCoordinates()[1],
+                    maxX: maxPoint.getCoordinates()[0],
+                    maxY: maxPoint.getCoordinates()[1],
+                };
+            }
+
```

Comment:
I spent way too much time trying to render the yellow bounding rect on openlayers like I did for leaflet. I gave up after a while. It'd be nice to have, but I think the example works OK without it.